doc/nrf/known_issues.rst

```diff
@@ -1048,6 +1133,33 @@ DRGN-13921: Directed advertising issues using RPA in TargetA
 DRGN-10367: Advertiser times out earlier than expected
   If an extended advertiser is configured with limited duration, it will time out after the first primary channel packet in the last advertising event.
 
+.. rst-class:: v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-11222: Master connections may disconnect prematurely if there are scheduling conflicts while doing a control procedure with an instant.
```

Comment:
1. Title should not end with period
2. Replace Master with "A Central may disconnect [...]" 

@wiba-nordic ?

Reply:
Good catch on the period.